sig-docs/migrated-from-wiki/roadmap-docs.md

```diff
@@ -8,7 +8,7 @@ Owners: @kubernetes/docs, @kubernetes/examples
 
 Labels: [kind/documentation](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Fdocumentation), [kind/example](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Fexample)
 
-We're currently planning a documentation site overhaul. Join kubernetes-dev@googlegroups.com to gain access to the proposals.
+We're currently planning a documentation site overhaul. Join dev@kubernetes.io to gain access to the proposals.
```

Comment:
this file does not need to be changed as its an archive so we should keep the history